src/diffusers/pipelines/audioldm2/pipeline_audioldm2.py

```diff
@@ -930,7 +948,7 @@ def __call__(
             latents = 1 / self.vae.config.scaling_factor * latents
             mel_spectrogram = self.vae.decode(latents).sample
         else:
-            return AudioPipelineOutput(audios=latents)
+            return ImagePipelineOutput(images=latents)
```

Comment:
If we're returning latents (effectively an image), using the `ImagePipelineOutput` is most suitable here IMO

Reply:
Even if that's the case I think `AudioPipelineOutput` is better to not introduce any confusion.

Reply:
Okay will revert!